packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -76,55 +66,19 @@ describe('HttpValidator', () => {
                 api_Key: 'ha',
               },
             },
-            config: { cors: false, mock: false, validate: { request: { headers: true, query: false, body: false } } },
+            config: { cors: false, mock: false, validateRequest: true, validateResponse: false },
           }),
         ).toEqual([]);
       });
     });
 
-    describe('only query validation is turned on', () => {
```

Comment:
A lot of this stuff got removed but I can't see from the diff that we're still covering query validation, are we?

Reply:
Yeah I need to check these again, I might have been too hasty in the removal